.jestrc

```diff
@@ -0,0 +1,3 @@
+{
+  "collectCoverage": true
```

Comment:
i hear that tests run a good bit slower when collectcoverage is always on -- what are your thoughts on running this as a separate command? something like this -- https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md#coverage-reporting